.github/workflows/scripts/check_for_new_files.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+  getAddedFiles: ({ github, context, core }) => {
+    const {
+      issue: { number: issue_number },
+      repo: { owner, repo }
+    } = context;
+
+    // Use the Github API to query for the list of files from the PR
+    github
+      .paginate(
+        'GET /repos/{owner}/{repo}/pulls/{pull_number}/files',
+        { owner, repo, pull_number: issue_number },
+        (response) => response.data.filter((file) => file.status === 'added')
+      )
+      .then((files) => {
+        // Save these values to the Github env
+        core.exportVariable('NEW_FILES', files);
+        core.exportVariable('NEW_FILES_COUNT', files.length);
+
+        console.log('New files: ', files);
+      });
+  },
+  validateCodeowners: async ({ github, context, fetch, ignore }) => {
+    const { NEW_FILES, CURRENT_BRANCH, CURRENT_REPO } = process.env;
+
+    const codeownersFile = `https://raw.githubusercontent.com/${CURRENT_REPO}/${CURRENT_BRANCH}/.github/CODEOWNERS`;
+
+    console.log('Fetching CODEOWNERS from: ', codeownersFile);
+
+    const response = await fetch(codeownersFile);
+    const body = await response.text();
+
+    // Filter out comments from CODEOWNERS file
+    const codeownersFilePatterns = body
+      .split('\n')
+      .filter((e) => !e.startsWith('#'))
+      .filter((e) => e.length > 1)
+      .map((e) => e.split(/\s+/)[0]);
+
+    console.log(codeownersFilePatterns);
```

Comment:
Did you mean to leave this console log? If so, let's make the log more descriptive. If not, let's remove it.